src/qt/overviewpage.cpp

```diff
@@ -152,6 +154,23 @@ void OverviewPage::handleOutOfSyncWarningClicks()
     Q_EMIT outOfSyncWarningClicked();
 }
 
+void OverviewPage::setPrivacy(bool privacy)
+{
+    m_privacy = privacy;
+    if (walletModel && walletModel->getOptionsModel()) {
```

Comment:
This condition looks unnecessary/unrelated, could add a comment maybe?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16432#issuecomment-633392179).